source/global-this.d.ts

```diff
@@ -0,0 +1,21 @@
+/**
+Type for `globalThis`. Extend to declare locally scoped global symbols.
+
+When defining a global symbol in a declaration file is inappropriate, it can be helpful to define an `interface` or `type` (say `Foo`) and then cast `globalThis` via code like `globalThis as unknown as Foo`.
+
+Instead of casting through `unknown`, you can extend `GlobalThis` and then directly cast `globalThis` to your local `interface`.
+
+@example
+```
+import type {GlobalThis} from 'type-fest';
+
+interface ExtraGlobals extends GlobalThis {
```

Comment:
Super nitpick, but I would prefer using a `type` here.

Reply:
Should we show both?

To be honest, I was surprised that this code-base makes interfaces prohibited and enforced that I write this as:

```ts
type ExtraGlobals = GlobalThis & {
	readonly GLOBAL_TOKEN: string;
};
```

At least for my organization, we have made the explicit decision to use `interface` rather than `type` for this sort of thing.

Reply:
I prefer to just show type. It's expected that a user can translate it to interface if they prefer that.

Reply:
Done.